src/ThreadedExecutor.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright (C) 2018 European Spallation Source, ERIC. See LICENSE file */
```

Comment:
Doesn't have the full header comment?

Reply:
Copied the file from another project that uses another style of header comment. Replaced it with one used here.